apps/web/src/components/Basenames/RegistrationFlow.tsx

```diff
@@ -91,204 +93,225 @@ export function RegistrationFlow() {
     }
   }, [basenameChain.id, searchParams, setSelectedName]);
 
+  const isDevelopment = process.env.NODE_ENV === 'development';
+
   return (
-    <main className={mainClasses}>
-      {/* 1. Brand & Search */}
-      <Transition
-        appear
-        show={isSearch}
-        className={classNames(
-          'absolute left-1/2 z-20 mx-auto w-full max-w-2xl -translate-x-1/2 -translate-y-1/2 transform  transition-opacity',
-          registrationTransitionDuration,
-          absoluteLayoutPosition,
-          {
-            'text-white': searchInputFocused,
-            'text-blue-600': searchInputFocused,
-          },
-        )}
-        enterFrom="opacity-0"
-        enterTo="opacity-100"
-        leaveFrom="opacity-100"
-        leaveTo="opacity-0"
-      >
-        <div className={classNames('relative bottom-full w-full pb-4', layoutPadding)}>
-          <RegistrationBrand />
-        </div>
+    <>
+      {isDevelopment && <RegistrationStateSwitcher />}
```

Comment:
`const RegistrationStateSwitcher = dynamic(() => import('../components/RegistrationStateSwitcher'), { ssr: false });`

If you want this to stick around for a while. (Though, I'm not generally a fan of dev UI, since it lets you skip out of the normal usage patterns and maybe not spot bugs.)

Reply:
done, we can def nuke this when we're done testing. For now this is for convenience / avoiding pushing the test dropdown by accident (which i've done a couple times)